### PR TITLE
Some random Swiftyness

### PR DESCRIPTION
**Pull request by aciidb0mb3r**
Some monotonic improvements

### CODE REVIEW

File: Sources/NIOChatServer/main.swift [hunk 1]
```diff
@@ -139,7 +140,7 @@ defer {
 // First argument is the program path
 let arguments = CommandLine.arguments
 let arg1 = arguments.dropFirst().first
-let arg2 = arguments.dropFirst().dropFirst().first
+let arg2 = arguments.dropFirst(2).dropFirst().first
```
Comment (ktoso) [inline Sources/NIOChatServer/main.swift]: Small one-off mistake? That drops 3 elements now but was previously dropping 2.

Should it be just:

```
arguments.dropFirst(2).first
```

?

Comment (aciidb0mb3r) [reply Sources/NIOChatServer/main.swift]: yikes, sorry

Comment (aciidb0mb3r) [reply Sources/NIOChatServer/main.swift]: Fixed!

Comment (ktoso) [reply Sources/NIOChatServer/main.swift]: 👍 :shipit:

**Comment by normanmaurer** [comment]
@swift-nio-bot test this please

File: .gitignore [hunk 2]
```diff
@@ -2,6 +2,6 @@
 /.build
 /Packages
 /*.xcodeproj
-Package.pins
+Package.resolved
```
Comment (ianpartridge) [inline .gitignore]: `Package.resolved` was already added in https://github.com/apple/swift-nio/commit/cb62f14df344e9bdab0dd7d133e00910de5aa463 so probably this is unnecessary.

**Comment by Lukasa** [comment]
@swift-nio-bot test this please

**Pull request merged**